Testing System/src/main/webapp/pages/singin.jsp

```diff
@@ -0,0 +1,25 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Регистрация</title>
```

Comment:
Please be consistent with the used language (don't mix Russian and English). 

All the fields that are language-sensitive can be set up using different locales. Since the project doesn't provide the "language change" option, it would be better to stick with only one language.